packages/jest-config/src/normalize.js

```diff
@@ -248,136 +236,78 @@ function normalize(config, argv) {
     config.collectCoverageOnlyFrom = collectCoverageOnlyFrom;
   }
 
-  if (!config.testRunner || config.testRunner === 'jasmine2') {
-    config.testRunner = require.resolve('jest-jasmine2');
-  } else {
-    config.testRunner = resolve(config, 'testRunner', config.testRunner);
-  }
-
   if (argv.env) {
     config.testEnvironment = argv.env;
   }
+};
 
-  if (config.testEnvironment) {
-    config.testEnvironment = getTestEnvironment(config);
-  }
-
-  let babelJest;
-  if (config.transform) {
-    const toString = Object.prototype.toString;
-    if (toString.call(config.transform) !== '[object Object]') {
-      const isString = typeof config.transform === 'string';
-      const exampleTransform = isString
-        ? config.transform
-        : '<rootDir>/node_modules/babel-jest';
-      throwConfigurationError(
-        `Configuration option \`${chalk.bold('transform')}\` ` +
-        'must be an object. Use it like this:\n\n' +
-        chalk.bold(
-          '  "transform": {\n' +
-          `    "^.+\\\\.js$": "${exampleTransform}"\n` +
-          '  }'
-        )
-      );
-    }
+function normalize(config: InitialConfig, argv: Object = {}) {
+  validate(config, VALID_CONFIG, DEPRECATED_CONFIG, {
+    footer: DOCUMENTATION_NOTE,
```

Comment:
I'm OK with comment. But what about `footNote`? I'm not a linguistic expert though 😄